app/boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840.zmk.yml

```diff
@@ -0,0 +1,10 @@
+file_format: "1"
```

Comment:
Should we add one of these for nrf5340dk_nrf5340 as well? I missed that with my PR to add support for it. (The board ID is nrf5340dk_nrf5340_cpuapp, so not sure what the file name should be.)

Reply:
Yes, probably should. It'd match the directory structure from the board, and the filename would match w/ the `_cpuapp` suffix.